content/advent-2015/test2doc-generate-your-api-docs.md

```diff
@@ -0,0 +1,294 @@
++++
+author = ["Sarah Adams"]
+date = "2015-12-14T22:04:31-08:00"
+series = ["Advent 2015"]
+title = "test2doc: Generate Your API Docs"
+
++++
+
+API docs are important. We all know that.
+They are also painful and tedious to maintain.
+When your docs aren't accurate, you get more questions. And everyone 
+loses time.
+
+I've always thought of unit tests as a great source of documentation. 
+But the non-Go engineers consuming my API don't often agree.
+
+Yet most of the same information is there:
+
+- request format:
+    - HTTP method
+    - URI
+    - query params
+    - request headers
+    - request body
+- response format:
+    - status code
+    - response headers
+    - response body
+
+<br>
```

Comment:
The `<br>` tag you're referring to in this particular case evens out the spacing before and after the list:
![screen shot 2015-12-10 at 10 14 24 am](https://cloud.githubusercontent.com/assets/2741356/11724111/f668b5d4-9f26-11e5-9d02-31156ef08680.png)